tests/cbmc/sources/make_common_datastructures.c

```diff
@@ -81,3 +81,29 @@ struct s2n_stuffer_reservation* cbmc_allocate_s2n_stuffer_reservation() {
     }
     return reservation;
 }
+
+struct s2n_array* cbmc_allocate_s2n_array() {
+    struct s2n_array *array = can_fail_malloc(sizeof(*array));
+    if(array != NULL) {
+        array->mem.size = nondet_uint32_t();
+        array->mem.allocated = nondet_uint32_t();
```

Comment:
Are these assignments necessary? Won't it already be nondet from the malloc?

Reply:
They're not, I removed them.